app/src/chat_engine.py

```diff
@@ -3,10 +3,12 @@
 from dataclasses import dataclass
 from typing import Sequence
 
+from sqlalchemy import Row
```

Comment:
So as not to expose `sqlalchemy.Row` in many files, I would replace `Sequence[Row[tuple[Chunk, float]]]` with `Sequence[ChunkData]` returned by `retrieve_with_scores()`, where `CheckData` is a new `@dataclass` with fields `chunk` and `score`, which provides the option of later adding `document` or `program` if needed. This suggestion (if desirable) can be done a separate PR.

Reply:
@KevinJBoyer Thoughts on the above comment? Shall I create a PR to implement it?

Reply:
Makes sense to me, though I'd have a slight preference for calling it ChunksWithScores or something like that vs. data. Alternatively, we could define the type directly like `ChunksWithScores = Row[tuple[Chunk, float]]` -- there might be disadvantages to that approach vs a dataclass though

Reply:
https://github.com/navapbc/labs-decision-support-tool/pull/32